app/src/App.tsx

```diff
@@ -31,6 +32,7 @@ export function AppContent() {
     <Provider theme={componentsTheme} mountGlobalStyles={false}>
       <RelayEnvironmentProvider environment={RelayEnvironment}>
         <GlobalStyles />
+        {window.Config.hasDbThreshold && <StorageAlert />}
```

Comment:
I think we should make this alerts section generic. Also since it's only specific to this scope, we should move it out of /components as it won't be re-used